### PR TITLE
Add drawPolygon for label2d handler

### PR DESCRIPTION
**Pull request by fyu**
This new function removes the redundant polygon creation code.

### CODE REVIEW

File: app/src/test/drawable/label2d_handler.test.ts [hunk 1]
```diff
@@ -484,34 +387,8 @@ test('2d polygons delete vertex and draw bezier curve', () => {
 
   // draw a polygon and delete vertex when drawing
   const canvasSize = new Size2D(1000, 1000)
-  label2dHandler.onMouseMove(new Vector2D(200, 100), canvasSize, -1, 0)
-  label2dHandler.onMouseUp(new Vector2D(200, 100), -1, 0)
-  label2dHandler.onMouseDown(new Vector2D(200, 100), -1, 0)
-  label2dHandler.onKeyDown(new KeyboardEvent('keydown', { key: 'd' }))
```
Comment (sinnyn) [inline app/src/test/drawable/label2d_handler.test.ts:490]: This key event missed.

Comment (sinnyn) [reply app/src/test/drawable/label2d_handler.test.ts:490]: I'll add a commit based on the original logic.

Comment (sinnyn) [reply app/src/test/drawable/label2d_handler.test.ts:490]: @fyu if the modification is fine, let me know to approve it.

Comment (fyu) [reply app/src/test/drawable/label2d_handler.test.ts:490]: The change looks great! I hope this new set of API can be used in future tests. Can you check why the tests fail?

Comment (sinnyn) [reply app/src/test/drawable/label2d_handler.test.ts:490]: Yes. I'm looking into it.

Comment (sinnyn) [reply app/src/test/drawable/label2d_handler.test.ts:490]: Fixed

Comment (fyu) [reply app/src/test/drawable/label2d_handler.test.ts:490]: Thanks!

**Pull request merged**